lib/Server.js

```diff
@@ -667,6 +669,11 @@ class Server {
     next();
   }
 
+  subscribeClientData(handle) {
+    this.app.on(CLIENT_EVENT, handle);
+    return () => this.app.off(CLIENT_EVENT, handle);
```

Comment:
Looks very misleading

Reply:
How do you want it then? :)

Reply:
Maybe `subscribe`/`unsubscribe`

Reply:
This pattern is pretty known though. Rxjs, redux and react hooks for instance uses it. Where you return the unsubscribe/teardown function. So the consumer don't need to store his handle himself.

Reply:
Do you like this better?

```js
subscribeClientData() {
  const { app }  = this;
  app.on(CLIENT_EVENT, handle);
  return function unsubscribeClientData() {
    app.off(CLIENT_EVENT, handle);
  };
}
```

Reply:
:+1:

Reply:
Fixed